src/api/txMining.js

```diff
@@ -73,6 +73,23 @@ const txMiningApi = {
       return Promise.reject(error);
     });
   },
+
+  /**
+   * Get health information for the tx-mining-service
+   *
+   * @return {Promise}
+   * @memberof ApiTxMining
+   * @inner
+   */
+  async getHealth() {
+    return new Promise((resolve, reject) => {
+      txMiningRequestClient(resolve).get(`health`).then((res) => {
+        resolve(res.data);
+      }, (res) => {
```

Comment:
Also here
```suggestion
      }, (err) => {
```

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-lib/pull/579/commits/9951ac7161a6ea17df875f0a8e2d9eb258d4c99a